static/src/javascripts/bootstraps/enhanced/common.js

```diff
@@ -120,6 +121,18 @@ const loadAnalytics = (): void => {
     }
 };
 
+const loadGoogleAnalytics = (): void => {
+    const handleGoogleAnalytics = (gaConsents: boolean): void => {
+        if (gaConsents && !window.ga.loaded) {
```

Comment:
`window.ga` is a function, so I can't quite see where the `loaded` property on `window.ga` is coming from?

Reply:
It's [set somehow by the JS we allow Google to run](https://stackoverflow.com/questions/29948923/how-to-detect-if-google-analytics-is-loaded-and-able-to-track-custom-events), it's not properly documented though so perhaps we should be setting our own variable somewhere for this instead. Let me try that.

Reply:
Ah ok, I was trying to search for it to no avail! I think it's worth setting a well described variable of our own yeah

Reply:
I did find it in some google documentation (as a code example for something unrelated). I think the risk of them removing it from their API silently is quite low but I suppose it's not much effort to do ourselves.